packages/js/components/src/experimental-tree-control/hooks/use-linked-tree.ts

```diff
@@ -9,7 +9,7 @@ import { useMemo } from 'react';
 import { Item, LinkedTree } from '../types';
 
 type MemoItems = {
-	[ value: Item[ 'value' ] ]: LinkedTree;
+	[ value: Item[ 'id' ] ]: LinkedTree;
```

Comment:
Is there a particular reason you changed to `id` instead of `value`?
Not a huge issue, but curious if there was a specific reason for this.

Reply:
I thought that since this is the first use case for TreeControl, and the equivalent data is called `id` on the `useCategorySearch` hook, it made sense to name it equally.